app/controllers/notices_controller.rb

```diff
@@ -26,7 +26,7 @@ def create
   # PATCH/PUT /notices/1
   def update
     if @notice.update(notice_params)
-      render json: @notice
+      render json: @notice, each_serializer: NoticeSerializer
```

Comment:
Cool! I wonder if there is some way to specify this once rather than in each method though. Also, I don't think this takes into account all of the parameters for json:api